pkg/commands/compute/setup/backend.go

```diff
@@ -98,9 +99,9 @@ func (b *Backends) Create() error {
 		if err != nil {
 			if !b.isOriginless() {
 				b.Spinner.StopFailMessage(msg)
-				err := b.Spinner.StopFail()
-				if err != nil {
-					return err
+				spinErr := b.Spinner.StopFail()
```

Comment:
I discovered a bunch of these issues where the actual API error was being accidentally overridden by a separate error and so (as a user) you wouldn't see an error related to what had gone wrong.